SUBGROUPS.md

```diff
@@ -132,7 +132,9 @@
 ## Use Case
 
 * Description & Responsibilities
-  * This group is has not started meeting yet.
+  * Create an open channel with the end users and potential deployment partners .
+  * Share the work being done by the groups and get feedback from the deployment partners .
+  * Encourage more deployment partners to contribute , take part and join .
 * Alignment to project goal(s) -- Workgroup leader: Please update this bullet

```

Comment:
This section ought to be updated as well. 
```suggestion
* Alignment to project goal(s) -- 
* By engaging with and seeking input from end users and deployment partners, the Use Case subgroup will help by validating the implementation examples and architectures and APIs being worked on by the other subgroups.
*This will also help build the open community to grow and include additional contributorsand members.

```